src/api/sql.js

```diff
@@ -3,6 +3,10 @@ var $ = require('jquery');
 var Mustache = require('mustache');
 var Promise = require('./promise');
 
+var errorMessages = {
```

Comment:
Something I learnt recently... if we use something like `NO_BOUNDS_ERROR_MESSAGE` here (instead of an object), it's easier for us (well, the linter) to catch unused variables.